guides/saas-retention-tracking-and-churn-with-powerbi.md

```diff
@@ -0,0 +1,102 @@
+---
+description: Sigma is a cloud-native analytics platform that uses a familiar spreadsheet interface to give business users instant access to explore and get insights from their cloud data warehouse. It requires no code or special training to explore billions of rows, augment with new data, or perform “what if” analysis on all data in real⁠-⁠time.
+---
+
+# Retention Tracking with Hydra and PowerBI
+
+## **Scope**
+The focus of this guide is to showcase the ease of integrating Hydra with [PowerBI]https://powerbi.microsoft.com/en-ca/) to develop a retention & churn-oriented dashboard on sample data within a Hydra database.
+
+To cater to a large audience, the data and KPIs were chosen to be as agnostic as possible. The data used is mock sales data created specifically for this demo, and the metrics are the following:
+
+* YoY Time Series Customer Retention Rate by Quarter 
+* YoY Time Series Customer Churn Rate by Quarter
+
+Your KPIs will differ, but this will hopefully serve as a foundation for the development of more metrics.
+
+<br>
+
+## **Setup**
+
+### Before You Start:
+- Ensure you have the details of your Hydra instance handy.
+- Download [PowerBI Desktop](https://powerbi.microsoft.com/en-us/desktop/).
+
+<br>
+
+### Building the Table with Hydra
+
+1. We will use sample data created specifically for this demo. The data comes in two .sql scripts: one to [create the table](), and one to [populate the table]() with the data. Retreive these two scripts and save them locally.
+
+2. Connect to your Hydra database via PSQL and run the two scripts with the following commands:
+
+	a. Grab the `psql` command from the Hydra dashboard and get into the `psql` console.
+
+	b. Run `saas_retention_ddl.sql` by entering: ` \i '<path_to_file_directory>/saas_retention_ddl.sql'`.
+
+    c. Run `saas_retention_data.sql` by entering ` \i '<path_to_file_directory>/saas_retention_data.sql'`.
+
+3. You should now have a table called `user_activity` populated with 96 monthly user records.
+
+<br>
+
+### Creating the Connection
+
+Now we can configure PowerBI to access our sample data in Hydra.
+
+1. Select `Get data` from the toolbar at the top. Click on `More`.
```

Comment:
@AmirZahre can we cut down on the user of code quotes to be only for code? just want to keep it consistent for areas where actual code or input is needed vs a button within the the platform. Happy to discuss on a quick video call as well.

Reply:
May have gotten a little carried away with those. Done!